CHANGELOG.md

```diff
@@ -18,6 +19,10 @@ Enables the use of pre-release builds of Omnisharp. Downloading a pre-release bu
   * "latest" - Use the latest CI build
   * `version` - Use a specific version of OmniSharp. Example: `1.29.2-beta.60`
 
+#### Editor
+
+* Updated the Omnisharp Status Bar to appear on left instead of right because it displays the project level info(which is a characteristic of the elements on the left side of the window) and removed ambiguities in its behavior.([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
```

Comment:
The description here is pretty difficult to understand. What ambiguities?

* Note that the rest of the change log uses present tense, rather than past tense.
* Rename `Omnisharp` to `OmniSharp`.
* Status Bar should not be capitalized. `OmniSharp status bar` is fine.
* There should be spaces between periods (`.`) and open parentheses (`(`)